### DISCUSSION TITLE
Can short Intervals cause problems?

### DISCUSSION THREAD

**Discussion by jlewi** [fluxcd/flux2#1155] [Q&A] [answered]
I had a Kustomization object with the reconcilation interval set to 1 minute and validation set to server.

Suddenly on a new commit the kustomization server started reporting validation failed; but the output & logs showed no validation errors. I also tried running `kubectl apply --dry-run=server` and that also showed no errors.

When I bumped the interval up to 5m. It started working.

Is this expected? 

Based on the docs it looks like the [Interval](https://toolkit.fluxcd.io/components/kustomize/kustomization/) is used to 

1. Determine the interval at which to reapply to correct for drift
2. Check for changes in the source

Therefore a short interval is needed in order to be responsive to git changes. Is this accurate? 

If so is there someway to cause Flux to respond immediately to changes to the source commit but have a longer interval for periodic reconciles?

**Comment by stefanprodan** [comment, marked as answer]
Please see https://toolkit.fluxcd.io/faq/#what-is-a-kustomization-reconciliation

The Kustomization interval has no relevance for source changes, a webhook or a git pull made by source-controller will trigger an immediate reconciliation.

**Reply by jlewi** [reply]
Thank you.
